ui/static/js/fortio_chart.js

```diff
@@ -439,3 +440,26 @@ function makeMultiChart(data) {
         }
       })
 }
+
+function runTestForDuration(durationInSeconds) {
+    document.getElementById('progressBarDiv').style.height = '1em'
+    document.getElementById('progressBarDiv').style.backgroundColor = 'black'
+    var startTimeMillis = Date.now()
+    var updatePercentage = null
+
+    updatePercentage = function () {
+	var barPercentage = 0
+	if (durationInSeconds < 0) { // infinite
+	    barPercentage = (Date.now() - startTimeMillis)/20 % 100 // 2 seconds
```

Comment:
can you add parenthesis (I know they aren't needed for precedence but for readability)
```
	    barPercentage = ((Date.now() - startTimeMillis) / 20) % 100 // 2 seconds cycle
```
took me a bit to see the modulo was outside/cycle

Reply:
done!